weblate/trans/vcs.py

```diff
@@ -630,6 +631,75 @@ def push(self, branch):
 
 
 @register_vcs
+class GithubRepository(GitRepository):
+
+    name = 'GitHub'
+
+    _cmd = 'hub'
+    _hub_config_file = os.path.join(os.path.expanduser('~'),
+                                    '.config/hub')
+    _hub_user_prefix = '- user: '
+
+    _hub_user = None
+    if 'GITHUB_USER' in os.environ:
+        _hub_user = os.environ['GITHUB_USER']
+    elif os.path.isfile(_hub_config_file):
```

Comment:
...and when having the setting in settings.py, I think this branch can be completely removed.